bin/iop

```diff
@@ -84,8 +89,9 @@ function render_kube_yaml() {
 }
 
 if [ "$1" == "" ] ; then
-    echo "iop NAMESPACE COMPONENT CONFIGDIR [-t|-d] [HELM_TEMPLATE_OPTIONS]"
+    echo "iop NAMESPACE COMPONENT CONFIGDIR [-c <cfgfile.yaml>] [-t|-d] [HELM_TEMPLATE_OPTIONS]"
```

Comment:
Why ? You can pass additional -f to override the global.yaml and values.yaml.
Not sure it's worth the complexity, any reason -f myglobal.yaml won't work ?

Reply:
Ok yeah... didn't recall I could pass an additional `-f ...`.  Will remove

Reply:
removing `-c ...`